<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DBRecordDlg</class>
 <widget class="QDialog" name="DBRecordDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>704</width>
    <height>83</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <property name="windowIcon">
   <iconset resource="../../../resources/resources.qrc">
    <normaloff>:/aplicacion/images/alephLogoIcon.ico</normaloff>:/aplicacion/images/alephLogoIcon.ico</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="widgetLayout"/>
   </item>
   <item>
    <widget class="QCheckBox" name="chkNavigateSavingChanges">
     <property name="whatsThis">
      <string>Si este checkbox está seleccionado, y usted ha modificado datos, éstos se guardarán automáticamente al pulsar los botones &quot;Anterior&quot;, &quot;Siguiente&quot;, &quot;Primero&quot; o &quot;Último&quot;. Si no está seleccionado, se le preguntará si desea guardar los datos modificados.</string>
     </property>
     <property name="text">
      <string>Guardar los datos modificados automáticamente al navegar por los registros</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frameButtons">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="lineWidth">
      <number>0</number>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="dbButtonsLayout">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pbEditScript">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="toolTip">
           <string>Edita el script que controla esta página</string>
          </property>
          <property name="statusTip">
           <string>Edita el script que controla esta página</string>
          </property>
          <property name="whatsThis">
           <string>Edita el script que controla esta página</string>
          </property>
          <property name="text">
           <string>Script</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>:/presupuestos/images/kate.png</normaloff>:/presupuestos/images/kate.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pbPrint">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="toolTip">
           <string>Abre el formulario de impresión de este registro. 
Podrá enviarlo también por correo electrónico (F9)</string>
          </property>
          <property name="statusTip">
           <string>Abre el formulario de impresión de este registro. \nPodrá enviarlo también por correo electrónico (F9)</string>
          </property>
          <property name="whatsThis">
           <string>Abre el formulario de impresión de este registro. 
Podrá enviarlo también por correo electrónico (F9)</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../../../resources/resources.qrc">
            <normaloff>:/generales/images/print.png</normaloff>:/generales/images/print.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
          <property name="shortcut">
           <string>F9</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pbHistory">
          <property name="statusTip">
           <string>Muestra el historial de cambios de este registro (F10)</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../../../resources/resources.qrc">
            <normaloff>:/generales/images/history.png</normaloff>:/generales/images/history.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
          <property name="shortcut">
           <string>F10</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pbSave">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="toolTip">
           <string>Guarda la información modificada de este registro. (F12)</string>
          </property>
          <property name="statusTip">
           <string>Guarda la información modificada de este registro. (F12)</string>
          </property>
          <property name="whatsThis">
           <string>Guarda la información modificada de este registro. (F12)</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../../../resources/resources.qrc">
            <normaloff>:/generales/images/save.png</normaloff>:/generales/images/save.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
          <property name="shortcut">
           <string>F12</string>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="toolTip">
           <string>Guarda el registro actual e inicia la navegación por uno nuevo (F11)</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../../../resources/resources.qrc">
            <normaloff>:/generales/images/saveAndNew.png</normaloff>:/generales/images/saveAndNew.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
          <property name="shortcut">
           <string>F11</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pbFirst">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="toolTip">
           <string>Ir al primer registro (Page Init)</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../../../resources/resources.qrc">
            <normaloff>:/generales/images/firstrecord.png</normaloff>:/generales/images/firstrecord.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
          <property name="shortcut">
           <string>Home</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pbPrevious">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="toolTip">
           <string>Ir al registro anterior (Retr. Pag.)</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../../../resources/resources.qrc">
            <normaloff>:/generales/images/previuosrecord.png</normaloff>:/generales/images/previuosrecord.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
          <property name="shortcut">
           <string>PgUp</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pbNext">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="toolTip">
           <string>Ir al registro siguiente (Avz. Pág)</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../../../resources/resources.qrc">
            <normaloff>:/generales/images/nextrecord.png</normaloff>:/generales/images/nextrecord.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
          <property name="shortcut">
           <string>PgDown</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pbLast">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="toolTip">
           <string>Ir al último registro (Page End)</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../../../resources/resources.qrc">
            <normaloff>:/generales/images/lastrecord.png</normaloff>:/generales/images/lastrecord.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
          <property name="shortcut">
           <string>End</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pbClose">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="toolTip">
           <string>Pulse aquí para cerrar esta ventana. (ESC)</string>
          </property>
          <property name="statusTip">
           <string>Pulse aquí para cerrar esta ventana.</string>
          </property>
          <property name="whatsThis">
           <string>Pulse aquí para cerrar esta ventana.</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../../../resources/resources.qrc">
            <normaloff>:/generales/images/close.png</normaloff>:/generales/images/close.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
          <property name="shortcut">
           <string>Esc</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../resources/resources.qrc"/>
  <include location="../../../resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
